src/messages/manifestjson.js

```diff
@@ -35,6 +35,16 @@ export const MANIFEST_FIELD_INVALID = {
   file: MANIFEST_JSON,
 };
 
+export const MANIFEST_BAD_PERMISSION = {
+  code: 'MANIFEST_BAD_PERMISSION',
+  legacyCode: null,
+  message: _('The permission type is unsupported.'),
+  description: _(singleLineString`Permissions must be strings; object
```

Comment:
Amended the message to be `Permissions must be strings. If a permission is an object, it's likely from a Chrome App Extension and will not work in Firefox. [...]`